docs/guide.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+---
+
+# Vitessce Link Detailed Guide
+
+**Thank you for using Vitessce Link\!**  
+
+This detailed guide is intended to help provide additional resources for the users of our Beta Launch. We will continue to update this document with information as needs arise.
+
+The Vitessce Link software is in-progress, and we will continue to develop and release additional features throughout the 12 month Beta period\!
+
+**If you have any questions about the Vitessce Link software or setting up your Meta Quest 3** please contact:
+
+[hidive@hms.harvard.edu](mailto:hidive@hms.harvard.edu)            
+**Subject**: Vitessce Link
```

Comment:
`mailto` links actually let us specify a default subject: https://www.w3docs.com/snippets/html/how-to-create-mailto-links.html

```suggestion
[hidive@hms.harvard.edu](mailto:hidive@hms.harvard.edu?subject=Vitessce Link)            
```